libs/providers/flagd/README.md

```diff
@@ -107,9 +107,8 @@ For general information on events, see the [official documentation](https://open
 
 ### Flag Metadata
 
-| Field   | Type   | Value                                             |
-| ------- | ------ | ------------------------------------------------- |
-| `scope` | string | "selector" set for the associated source in flagd |
+Flag metadata is a set of key-value pairs that can be associated with a flag.
```

Comment:
Reference the docs: https://deploy-preview-1505--polite-licorice-3db33c.netlify.app/reference/flag-definitions/#metadata